fars/booking/static/js/tablet.js

```diff
@@ -0,0 +1,100 @@
+function updateTime() {
+  var hours = new Date().getHours();
```

Comment:
```suggestion
  const hours = new Date().getHours();
```

Maybe the usage of `const` instead of `var` would make it clearer that these are not variables?
Same for other parts.